x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/use_host_isolation_status.tsx

```diff
@@ -59,7 +59,7 @@ export const useHostIsolationStatus = ({
           return;
         }
 
-        if (isMounted && error.body.statusCode === 404) {
+        if (isMounted && error.body.statusCode === 400) {
```

Comment:
Yeah, a `400` is not good and I would like to see the Server error that caused it. The `404` should be a controlled error (one that we explicitly return), but the 400 implies something else (or maybe the API is not returning the correct code).